creativecloud/blocks/change-bg/change-bg.js

```diff
@@ -0,0 +1,168 @@
+      defaultBgSrc: `${assetsRoot1}/tablet/defaultBg.webp`,
+      tryitSrc: `${assetsRoot}/tryit.svg`,
+      tryitText: 'Try it',
+      groups: [
+        {
+          name: 'Remove Background',
+          iconUrl: `${assetsRoot1}/remove-background-icon.svg`,
+        },
+        {
+          name: 'Change Photo',
+          iconUrl: `${assetsRoot1}/change-photo-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/tablet/photo1.jpeg`,
+              swatchSrc: `${assetsRoot}/photo-submenu-1.jpeg`,
+            },
+          ],
+        },
+        {
+          name: 'Change Color',
+          iconUrl: `${assetsRoot1}/change-color-icon.svg`,
+          options: [
+            { src: '#31A8FF' },
+          ],
+        },
+        {
+          name: 'Change Pattern',
+          iconUrl: `${assetsRoot1}/change-pattern-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/tablet/pattern1.jpeg`,
+              swatchSrc: `${assetsRoot}/pattern-submenu-1.jpeg`,
+            },
+          ],
+        },
+      ],
+    },
+    mobile: {
+      marqueeTitleImgSrc: `${assetsRoot1}/mobile/everyonecanphotoshop.webp`,
+      talentSrc: `${assetsRoot1}/mobile/yogalady.webp`,
+      defaultBgSrc: `${assetsRoot1}/mobile/defaultBg.webp`,
+      tryitSrc: `${assetsRoot}/tryit.svg`,
+      tryitText: 'Try it',
+      groups: [
+        {
+          name: 'Remove Background',
+          iconUrl: `${assetsRoot1}/remove-background-icon.svg`,
+        },
+        {
+          name: 'Change Photo',
+          iconUrl: `${assetsRoot1}/change-photo-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/mobile/photo1.jpg`,
+              swatchSrc: `${assetsRoot}/photo-submenu-1.jpg`,
+            },
+          ],
+        },
+        {
+          name: 'Change Color',
+          iconUrl: `${assetsRoot1}/change-color-icon.svg`,
+          options: [
+            { src: '#31A8FF' },
+          ],
+        },
+        {
+          name: 'Change Pattern',
+          iconUrl: `${assetsRoot1}/change-pattern-icon.svg`,
+          options: [
+            {
+              src: `${assetsRoot}/mobile/pattern1.jpg`,
+              swatchSrc: `${assetsRoot}/pattern-submenu-1.jpg`,
+            },
+          ],
+        },
+      ],
+    },
+  };
+}
+
+export default async function init(el) {
+  assetsRoot = el.querySelector(':scope > div > div').innerHTML;
+  console.log('el', assetsRoot);
+  // prefetch the mobile background image
+  if (matchMedia('screen and (max-width: 599px)').matches) {
+    const img1 = new Image();
+    img1.fetchPriority = 'high';
+    img1.src = `${assetsRoot1}/mobile/everyonecanphotoshop.webp`;
+    const img2 = new Image();
+    img2.fetchPriority = 'high';
+    img2.src = `${assetsRoot1}/mobile/yogalady.webp`;
+    const img = new Image();
+    img.fetchPriority = 'high';
+    img.src = `${assetsRoot1}/mobile/defaultBg.webp`;
+  }
```

Comment:
Just wanted to confirm if we need this block. This will recudce the load delay for the LCP by loading the LCP image early but it is not getting rendered by the library till the layers are processed and this early load will add to the render delay time and hence will not give much improvement.